### PR TITLE
[WWST-6110, WWST-6146] EZEX fingerprint of zigbee-switch.groovy file.

### PR DESCRIPTION
**Pull request by juhaki**
Please add the EZEX fingerprint of the zigbee-switch.groovy file for the EZEX switch ( component 1 )

		// EZEX
		fingerprint profileId: "0104", deviceId: "0100", endpoint: "01", inClusters: "0000, 0003, 0004, 0006", outClusters: "0006, 000A, 0019", manufacturer: "EZEX", model: "E220-KR1N0Z0-HA", deviceJoinName: "EZEX Switch" // 1

### CODE REVIEW

File: devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy [hunk 1]
```diff
@@ -82,6 +82,9 @@ metadata {
 		// Third Reality
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0019", manufacturer: "Third Reality, Inc", model: "3RSS008Z", deviceJoinName: "RealitySwitch Plus"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0019", manufacturer: "Third Reality, Inc", model: "3RSS007Z", deviceJoinName: "RealitySwitch"
+		
+		// EZEX
```
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy]: @juhaki please move this section up so it's in alphabetical order by manufacturer

File: devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy [hunk 2]
```diff
@@ -82,6 +82,9 @@ metadata {
...
+		fingerprint profileId: "0104", deviceId: "0100", endpoint: "01", inClusters: "0000, 0003, 0004, 0006", outClusters: "0006, 000A, 0019", manufacturer: "EZEX", model: "E220-KR1N0Z0-HA", deviceJoinName: "EZEX Switch"
```
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy]: You can remove the deviceId and endpoint to more closely match the format used by the other fingerprints in this file

File: devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy (hunk 1 above)
Comment (juhaki) [reply devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy]: Ok. I will patch to this about alphabetical order move this section.

File: devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy (hunk 2 above)
Comment (juhaki) [reply devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy]: After the testing by removed deviceId & endpoint items.
I will remove the deviceId & endpoint.

**Review by tpmanley — APPROVED** [review]

**Pull request merged**